int8 drone_id
crazyflie_trajectory_manager/ConfigTrajectoryBoolPayload payload
---
bool proxy_success
bool result